### ISSUE TITLE
onSelectionChanged not fired in excel on the web in same cell but is fired in excel desktop

### ISSUE THREAD

**Issue by didu9898** [OfficeDev/office-js#4691]
onSelectionChanged not fired in excel online but is fired in excel desktop if address param in selectRange() is same as current address
# Provide required information needed to triage your issue

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Office on the web
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: _____
* Operating System: Windows
* Browser (if using Office on the web): Chrome
 
## Expected behavior
onSelectionChanged should be fired in excel web same as excel desktop i.e. behavior should be same

## Current behavior
For e.g. I am currently on cell A1 and I call selectCell('A1'), onSelectionChange event is fired in excel desktop but it is not fired in web. 

## Steps to reproduce
```
$("#run").on("click", () => tryCatch(run));
async function run() {
  await Excel.run(async (context) => {
    const sheet = context.workbook.worksheets.getActiveWorksheet();
    sheet.onSelectionChanged.add((args) => {
      console.log('Selection changed event fired.');
      return Promise.resolve();
    });
    // Register the onSelectionChanged event handler
    await context.sync();
    const range = sheet.getRange('A1');
    range.select();
    await context.sync();
  });
}
```

/** Default helper for invoking an action and handling errors. */
async function tryCatch(callback) {
  try {
    await callback();
  } catch (error) {
    // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
    console.error(error);
  }
}
`

This is ScriptLab code. Manually select any other cell say B2. Then click "run" in ScriptLab. There will be a console log 'Selection changed event fired' as expected and A1 cell will be selected. Now again click "run" in ScriptLab. In excel desktop, the same console log will appear but in web, there will be no console log because in web this event is not fired when we try to select same cell as the current cell.

## Link to live example(s)

1. ______
2. ______
3. ______

# Provide additional details

1. ______
2. ______
3. ______

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by shanshanzheng-dev** [comment]
Hi @didu9898 Thanks for reporting this issue. Seem I can not repro your issue, could you try to clean cache and try this API again?

https://github.com/user-attachments/assets/0534ef35-af68-4442-bbdd-2312b6048aa3

**Comment by didu9898** [comment]
The issue is happening when I try to select the same cell as current cell. In my example code, selectRange('A1') is called so may be stay in cell A1 and then click "Run".

**Comment by shanshanzheng-dev** [comment]
Hi @didu9898 Thanks for confirming with us. We'll be looking into this issue and we'll report back if we have a suggestion for you.

**Comment by didu9898** [comment]
@shanshanzheng-dev were you able to reproduce the issue?

**Comment by shanshanzheng-dev** [comment]
Hi @didu9898 Yes, If my understanding is right, when I selected A1 cell, the onSelectionChanged will be not fired. we'll be looking into the root cause. We unfortunately have no timelines to share at this point.

**Comment by didu9898** [comment]
Hi @shanshanzheng-dev thank you for confirming.

**Comment by didu9898** [comment]
Hello @shanshanzheng-dev, is there any updates on this? Can you suggest any workaround other than selecting other cell and then selecting back the same cell?

**Comment by XuanZhouMSFT** [comment]
Hi @didu9898, we already release a fix on version > 16.0.17928 in Windows and version > 16.88 on MAC,  please check whether this still repro on your side. Thanks so much. Please feel free to let us know if you still have other issues.
